src/apps/content-editor/src/app/views/ItemEdit/Content/ActionsDrawer/ActionsDrawer.js

```diff
@@ -107,22 +108,29 @@ export default function ActionsDrawer(props) {
 
       <div className={styles.Spacer}></div>
       <Button
-        className={cx(
-          styles.ActionsDrawerButton,
-          ui.contentActionsHover || ui.contentActions
-            ? styles.ActionsDrawerButtonHover
-            : ""
-        )}
+        variant="contained"
         data-cy="ActionsButton"
         title="Open for additional file information"
         onClick={() => {
           dispatch(actions.setContentActions(!ui.contentActions));
         }}
+        sx={{
```

Comment:
Why the use of inline styles?

Reply:
Inline styles is the way we are moving forward with the migration as discussed. This follows our Marketing site and MUI conventions. 

CC: @agalin920